core/eolearn/tests/test_core_tasks.py

```diff
@@ -387,6 +390,31 @@ def test_map_features(test_eopatch):
         MapFeatureTask(f_in, f_out)
 
 
+@pytest.mark.parametrize(
+    "feature,  task_input",
+    [
+        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): [2, 4, 8]}),
+        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): [2]}),
+        (
+            (FeatureType.DATA, "REFERENCE_SCENES"),
+            {(FeatureType.DATA, "B01"): [0], (FeatureType.DATA, "B02"): [1], (FeatureType.DATA, "B02 & B03"): [1, 2]},
+        ),
+        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): []}),
+    ],
+)
+def test_explode_bands(
+    test_eopatch: EOPatch,
+    feature: FeatureType,
+    task_input: Dict[Tuple[FeatureType, str], Iterable[int]],
+):
+    move_bands = ExplodeBandsTask(feature, task_input)
+    patch = move_bands(test_eopatch)
+    assert all(new_feature in patch for new_feature in task_input.keys())
```

Comment:
no need to use `.keys()` when using iteration over lists (keys are iterated over by default)